File: utilities/src/test/java/com/toocol/termio/utilities/log/TermioLoggerTest.java
package com.toocol.termio.utilities.log;

import org.junit.jupiter.api.Test;

/**
 * @author ZhaoZhe (joezane.cn@gmail.com)
 * @date 2022/6/28 15:01
 */
class TermioLoggerTest {
    @Test
    void testLog() {
        LoggerFactory.init(null);
        Logger logger = LoggerFactory.getLogger(TermioLoggerTest.class);
        logger.info("Testing log {}", 1);
    }
}

File: utilities/src/test/java/com/toocol/termio/utilities/log/TermioLoggerTest.kt
package com.toocol.termio.utilities.log

import org.junit.jupiter.api.Test

/**
 * @author ZhaoZhe (joezane.cn@gmail.com)
 * @date 2022/6/28 15:01
 */
internal class TermioLoggerTest {
    @Test
    fun testLog() {
        LoggerFactory.init()
        val logger = LoggerFactory.getLogger()
        logger.info(javaClass.simpleName, "Testing log {}", 1)
    }
}